test/integration-test/src/tests/smoke.rs

```diff
@@ -4,8 +4,14 @@ use aya::{
     Bpf, BpfLoader,
 };
 
+use crate::utils::Netns;
+
 #[test]
 fn xdp() {
+    Netns::new().exec(do_xdp)
+}
+
+fn do_xdp() {
```

Comment:
```suggestion
    Netns::new().exec(|| {
```
You can use a closure to avoid creating a new function.

Reply:
Done in a956e8f